tools/rocksdb-stats/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use nearcore::{get_default_home, get_store_path};
+use serde::Serialize;
+use std::collections::HashMap;
+use std::process::Command;
+
+#[derive(Serialize, Debug)]
+struct Data {
+    col: String,
+    entries: u64,
+    estimated_table_size: u64,
+    // Total size of all keys in bytes.
+    raw_key_size: u64,
+    // Total size of all values in bytes.
+    raw_value_size: u64,
+}
+
+// SST file dump keys we use to collect statistics.
+const SST_FILE_DUMP_LINES: [&str; 5] =
+    ["column family name", "# entries", "(estimated) table size", "raw key size", "raw value size"];
+
+impl Data {
+    pub fn from_sst_file_dump(lines: &[&str]) -> Self {
+        // Mapping from SST file dump key to value.
+        let mut values: HashMap<&str, &str> = Default::default();
+
+        for line in lines {
+            let split_line: Vec<&str> = line.split(':').collect();
+            if split_line.len() < 2 {
+                continue;
+            }
+            let line = split_line[0].trim();
+            let value = split_line[1].trim();
+
+            for sst_file_line in SST_FILE_DUMP_LINES {
+                if line == sst_file_line {
+                    if values.contains_key(line) {
+                        panic!(
+                            "Line {} was presented twice and contains values {} and {}",
+                            line,
+                            values.get(line).unwrap(),
+                            value
+                        );
+                    } else {
+                        values.insert(line, value);
+                    }
+                }
+            }
+        }
+
+        Data {
+            col: String::from(values.get(SST_FILE_DUMP_LINES[0]).unwrap().clone()),
+            entries: values.get(SST_FILE_DUMP_LINES[1]).unwrap().parse::<u64>().unwrap(),
+            estimated_table_size: values
+                .get(SST_FILE_DUMP_LINES[2])
+                .unwrap()
+                .parse::<u64>()
+                .unwrap(),
+            raw_key_size: values.get(SST_FILE_DUMP_LINES[3]).unwrap().parse::<u64>().unwrap(),
+            raw_value_size: values.get(SST_FILE_DUMP_LINES[4]).unwrap().parse::<u64>().unwrap(),
+        }
+    }
+
+    pub fn merge(&mut self, other: &Self) {
+        self.entries += other.entries;
+        self.estimated_table_size += other.estimated_table_size;
+        self.raw_key_size += other.raw_key_size;
+        self.raw_value_size += other.raw_value_size;
+    }
+}
+
+fn main() {
+    let home_dir = get_default_home();
+    let store_dir = get_store_path(&home_dir);
+    let mut cmd = Command::new("sst_dump");
+    cmd.arg(format!("--file={}", store_dir.to_str().unwrap()))
```

Comment:
Would the following work? 

```suggestion
    cmd.arg("--file").arg(&store_dir)
```

? Path are not strings, better not to call `to_str` and the like.